lms/static/scripts/frontend_apps/config.js

```diff
@@ -30,6 +30,7 @@ import { createContext } from 'preact';
  * @prop {string} userid
  * @prop {string} LISResultSourcedId - Unique outcome identifier
  * @prop {string} LISOutcomeServiceUrl - API URL for posting outcome results
+ * @prop {string} lmsId
```

Comment:
Attn @marcospri — I need help explaining what this property actually is :)

Reply:
Can you ping Marcos again to find out and update the docs here.

Reply:
@marcospri I am pinging you here :)

Reply:
https://github.com/hypothesis/lms/pull/3441#discussion_r760857163

userid - The unique identifier of the user on H
lmsId - The unique identifier of the user on the corresponding LMS (Canvas, Blackboard...)

Hope that helps :+1: